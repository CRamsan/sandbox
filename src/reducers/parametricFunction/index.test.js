import reducer from '.'

const initialState = {
  equation: 'x^2 + y^2',
}

describe('parametricFunction', () => {
  it('should return the initial state', () => {
    const next = reducer(initialState, {})
    expect(next).toEqual(initialState)
  })

  it('PARAMETRIC_FUNCTION_SET_EQUATION', () => {
    const next = reducer(initialState, { type: 'PARAMETRIC_FUNCTION_SET_EQUATION', equation: 'x^2 - y^2' })
    expect(next).toMatchSnapshot()
  })
})
