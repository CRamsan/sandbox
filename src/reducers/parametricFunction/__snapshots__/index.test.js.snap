// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parametricFunction PARAMETRIC_FUNCTION_SET_EQUATION 1`] = `
Object {
  "equation": "x^2 - y^2",
}
`;
